From 783e28fa6c56cf38b5380cb8495f56234c874ac8 Mon Sep 17 00:00:00 2001
From: "guoyin.chen" <guoyin.chen@freescale.com>
Date: Fri, 5 Aug 2011 18:47:00 +0800
Subject: [PATCH 19/43] ENGR00154415 Remove the net.dns prop setting in
 rild/pppd

framework will handle the net.dns prop setting.
rild only need to make sure the net.ppp0.dns1 been set

Signed-off-by: guoyin.chen <guoyin.chen@freescale.com>
---
 reference-ril/reference-ril.c | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/reference-ril/reference-ril.c b/reference-ril/reference-ril.c
index e18917a..279e992 100755
--- a/reference-ril/reference-ril.c
+++ b/reference-ril/reference-ril.c
@@ -2010,7 +2010,7 @@ static void requestSetupDataCall(void *data, size_t datalen, RIL_Token t)
 	    if(!strncmp(buffer, "up", strlen("up")) || !strncmp(buffer, "unknown", strlen("unknown"))) {
 		// Should already get local IP address from PPP link after IPCP negotation 
 		// system property net.ppp0.local-ip is created by PPPD in "ip-up" script 
-                local_ip[0] = 0;
+        local_ip[0] = 0;
 		property_get("net.ppp0.local-ip", local_ip, "");
 		if(!strcmp(local_ip, "")) {
 		    LOGW("PPP link is up but no local IP is assigned. Will retry %d times after %d seconds", \
@@ -2018,6 +2018,7 @@ static void requestSetupDataCall(void *data, size_t datalen, RIL_Token t)
 		} else {
 		    LOGD("PPP link is up with local IP address %s", local_ip);
 		    // other info like dns will be passed to framework via property (set in ip-up script)
+            response[1] = "ppp0";
 		    response[2] = local_ip;  
 		    // now we think PPPd is ready
 		    break;	
-- 
1.8.0

